src/lib/mongoDriver/mongoConnectionPool.cpp

```diff
@@ -329,54 +330,76 @@ static std::string composeMongoUri
 {
   // Compose the mongoUri, taking into account all information
 
-  std::string uri = "mongodb://";
+  std::string uri;
 
-  // Add auth parameter if included
-  if (strlen(username) != 0 && strlen(passwd) != 0)
+  if (strlen(dbURI) != 0)
```

Comment:
If `-dbUri` is used at the same time some of the old parameters (`-dbhost`, etc.) I understand that `-dbUri` will take preference.

That's fine, but such case of combining old and new CLI would be probably a misconfiguration that should be at least warned to the user (or maybe even better, raise a LM_X so the user have the chance to fix the configuration).

Reply:
Fixed in 477566c3736507c8afd2009317bd7b9aec67bc61